airflow/settings.py

```diff
@@ -237,9 +237,7 @@ def configure_orm(disable_connection_pool=False, pool_class=None):
         session = Session()
         try:
             result = session.execute(
-                sqlalchemy.text(
-                    "SELECT is_read_committed_snapshot_on FROM sys.databases WHERE name=:database_name"
-                ),
+                sa.text("SELECT is_read_committed_snapshot_on FROM sys.databases WHERE name=:database_name"),
```

Comment:
I think `import sqlalchemy as sa` it is not common pattern if we compare to `import pandas as pd`, `import numpy as np` and `import multiprocessing as mp`, I know only one place where this pattern uses - [`alembic`](https://alembic.sqlalchemy.org/en/latest/tutorial.html#running-our-second-migration), in all other places include SQLAlchemy's documentation common import pattern is `from sqlalchemy import something`.

Particular in this file we've already imported `text` from `sqlalchemy`

Reply:
While `alembic` scripts are very sqlalchemy-centric, having `from sqlalchemy import …` words like `text` or `delete` in a large file looks a bit confusing to me. I'm fine with either though, but would prefer one way of doing it.

Reply:
Second with @Taragolis 's thoughts. Or should we use `import sqlalchemy` instead of `from sqlalchemy import ...`